packages/mobile/src/navigator/Headers.v2.tsx

```diff
@@ -97,6 +100,12 @@ export const headerWithCancelButton: StackNavigationOptions = {
   headerLeft: () => <CancelButton />,
 }
 
+export const headerWithCloseButton: StackNavigationOptions = {
+  ...emptyHeader,
+  headerLeft: () => <TopBarIconButton icon={<Times />} onPress={navigateBack} />,
+  headerLeftContainerStyle: { paddingLeft: 20 },
```

Comment:
my understand is that paddings are 8, 12, 16 or 24. 

also how about having this style be defined in a Stylesheet.create?

Reply:
Yes it should be one of these sizes, however it felt a bit too far on the right in the iOS modal style.
We can revisit later.